packages/graphback-runtime-mongodb/src/MongoDBDataProvider.ts

```diff
@@ -92,11 +93,11 @@ export class MongoDBDataProvider<Type = any, GraphbackContext = any> implements
     const { idField } = getDatabaseArguments(this.tableMap, filter);
     // TODO MongoDB should use advanced filter with JSON scalar defined as InputType
     if (filter[idField.name]) {
-      const query = this.db.collection(this.collectionName).
+      const query = mquery(this.db.collection(this.collectionName)).
         find({ _id: new ObjectId(filter[idField.name]) });
       dbResult = await this.usePage(query, page);
     } else {
-      const query = this.db.collection(this.collectionName).find(filter);
+      const query = mquery(this.db.collection(this.collectionName)).find(filter);
```

Comment:
Me thinking maybe we can have mquery as property:

`mquery(this.db.collection(this.collectionName))` => `this.mcollection `

Reply:
Agreed :+1:  it's a bit of a mouthful